api.swagger.yaml

```diff
@@ -1709,6 +1789,59 @@ paths:
         '422':
           description: JSON schema validation failure
       description: Report a post as the current user.
+  '/manuals':
+    get:
+      description: Get all the manual (root) pages
+      tags:
+        - manuals
+      parameters:
+        - $ref: '#/components/parameters/offsetParam'
+        - $ref: '#/components/parameters/limitParam'
+      responses:
+        '200':
+          description: Returns all the root manual pages
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  count:
+                    description: Total count manuals
+                    type: integer
+                    example: 248
+                  results:
+                    description: An array of the manual pages
+                    type: array
+                    items:
+                      allOf:
+                        - $ref: '#/components/schemas/manual-page-listing-schema'
+  '/manuals/{manual_page_id}':
+    get:
+      description: Get a manual page, including the root manual page. Offset and limit refer to child pages
+      tags:
+        - manuals
+      parameters:
+        - description: The Invotra UUID of a manual page.
+          in: path
+          name: manual_page_id
+          required: true
+          schema:
+            type: string
+            format: uuid
+            example: '01234567-89ab-cdef-1234-56789abcdef0'
+        - $ref: '#/components/parameters/offsetParam'
+        - $ref: '#/components/parameters/limitParam'
+      responses:
+        '200':
+          description: Returns a single complete manual or manual page object.
```

Comment:
See comment above - should this endpoint return manuals as well as manual pages? (I thought it was just meant to be manual pages but I may be wrong)